### DISCUSSION TITLE
How to generate SQL statement using Presto parser

### DISCUSSION THREAD

**Discussion by rahulcodewiz** [prestodb/presto#15043] [General] [answered]
I am trying SQL parser to alter existing SQL statement and generate it back? My usecase is-

1. Parse existing SQL statement
2. Append new columns and where clause conditions.
3. Regenerate SQL Statement.

I am unable to figureout how to achieve step 3. Any suggestions would be a great help. Thanks

**Comment by rongrong** [comment, marked as answer]
Take a look at `SqlFormatter.java`

**Reply by rahulcodewiz** [reply]
It worked.
